### ISSUE TITLE
"Display Text" button opens and then closes immediately on the tab "Dropped Files"

### ISSUE THREAD

**Issue by anykeypress** [kevoreilly/CAPEv2#432]
## This is opensource and you getting __free__ support so be friendly!
* Free support from doomedraven ended, no whiskey no support. For something he updated the documentation :)

# Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (for multi-repository projects)
- [x] I'm have read all configs with all optional parts

# Expected Behavior
Display the (text) content of the dropped file selected.

# Current Behavior

"Display Text" button opens and then closes immediately on the tab "Dropped Files". If multiple dropped files are present, sometimes the text contents are displayed correctly, but the others are opened and closed immediately.

# Failure Information (for bugs)
This also happens on the public cape sandbox (https://capesandbox.com/analysis/115914/) is a good example.

## Steps to Reproduce

1. Submit a sample that has known dropped files
2. Select the tab "Dropped Files)
3. If multiple entries are present, sometime the firt one is shown normally, but subsequent ones are closed immediately

## Context

| Question         | Answer
|------------------|--------------------
| Git commit       | 783785d3081ad77269334725abf71b3bc8ececfe
| OS version       | Ubuntu 20.04, Windows 7 x64, Firefox Browser v. 85, also in the actual Chrome vesion

## Failure Logs

I can't see anything in the web log, perhaps it's a glitch in the Browser?

**Comment by doomedraven** [comment]
i saw that behavior, that has something to do with bootstrap js, but i didn't have time to investigate, and nobody of us is webdev

**Comment by ClaudioWayne** [comment]
Did you remove it?
![image](https://user-images.githubusercontent.com/35531629/107372352-3a198600-6add-11eb-8aeb-fc290acceef7.png)

**Comment by doomedraven** [comment]
yes bcz that is for bootstrap3 and bootstrap.min.js has it embed

**Comment by anykeypress** [comment]
@doomedraven , I have found a "work.around" for this bug. I've done a bit of research on the internet and cannot find anything specifically to rectify it... however this works for me...

In the file 
`/web/templates/analysis/dropped/index.html`
there are two instances of
`&emsp;<a class="btn btn-secondary btn-small" data-toggle="collapse" data-target="#{{file.sha256}}">Display Text</a>`
these should be replaced with
`&emsp;<a class="btn btn-secondary btn-small" data-toggle="collapse" data-target="#DisplayText">Display Text</a>`

also the instance
`<tr class="collapse" id="{{file.sha256}}">`
needs to be replaced with
`<tr class="collapse" id="DisplayText">`

basically all this does is remove the "variable" and use a hardcoded id for the text area to be displayed when using the drop down button. I don't know if this is acceptable for you guys... the only other thing I can think of is replacing the version of bootstrap... but that would proably be a major undertaking...

**Comment by doomedraven** [comment]
thank you @anykeypress, but that is the problem, if we have few files that can show content? then it won't work, but at least it sound like the variable is the problem

**Comment by anykeypress** [comment]
@doomedraven , why would that be a problem... or do you thin it also affects the other file types to display? As far as I have looked through the code, the only type that is displayed via "accordian / expanable" button is the type "Text"... (or {{file.sha256}}, the other types are just displayed... just my thoughts, ignore if inappropriate.

**Comment by doomedraven** [comment]
Bcz if you have more than 1 file with display text, target id is the same
and that is wrong

**Comment by doomedraven** [comment]
fixed here https://github.com/kevoreilly/CAPEv2/commit/cc02cc7e1000245c18564f6133cb50d4f8749d94
 thanks @anykeypress to help spot problem

**Comment by anykeypress** [comment]
hi @doomedraven , i did test that possibility (multiple text based files to display) and found no problems (except that both text areas opend / closed when multiple files where present)... example would be [this...](https://capesandbox.com/analysis/116215/) althouhg I got more dropped files then the sample on the public cape...
But whatever, your solution is ore elegant :-)

**Comment by doomedraven** [comment]
Hello, well is not more elegant, is just correct one, as each button should
show their text and not all
